### ISSUE TITLE
Add support for querying blackmap.

### ISSUE THREAD

**Issue by higuoxing** [greenplum-db/diskquota#78]
Currently, diskquota doesn't support querying blackmap items. It might be good to have one which helps engineer debug the extension and helps user view blocked queries.

e.g.,

```sql
SELECT diskquota.get_black_map();
  target oid  |  databaseoid  |  tablespaceoid  |  targettype  
---------------------------------------------------------------
  some_oid    |  some_oid     |  some_oid       |   ROLE_QUOTA
...
```

**Comment by xuebinsu** [comment]
This can be achieved by querying `diskquota.table_size` and `diskquota.quota_config`.

**Comment by higuoxing** [comment]
Resolved in #89
